src/Tracy/Logger/Logger.php

```diff
@@ -163,7 +163,6 @@ protected function sendEmail($message): void
 
 		if (
 			$this->email
-			&& $this->mailer
```

Comment:
this is wrong fix, instead the type of the `$mailer` property should be changed to `callable|null`

Reply:
Ok, now I understand. In original implementation property `$mailer` was setted by `__construct` in all cases. But property is public and can be changed.

I reverted condition here and change property definition:

```php
/** @var callable|null handler for sending emails */
public $mailer;
```